application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -154,20 +153,22 @@ public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
 
     private void transferFlow(ModalInteractionEvent event, String channelId, String authorId,
             String messageId) {
-
         event.getJDA()
             .retrieveUserById(authorId)
             .flatMap(fetchedUser -> createForumPost(event, fetchedUser))
             .flatMap(createdforumPost -> dmUser(event.getChannel(), createdforumPost,
                     event.getGuild()))
             .flatMap(dmSent -> deleteOriginalMessage(event.getJDA(), channelId, messageId))
             .queue();
+
+        event.getHook().sendMessage("Transferred 👍").queue();
```

Comment:
You could do something like this here.
``
```suggestion
         .flatMap(dmSent -> deleteOriginalMessage(event.getJDA(), channelId, messageId))
          .flatMap(deleted->event.getHook().sendMessage("transferred"))
            .queue();
```

Reply:
> You could do something like this here.

tried that wasn't working

Reply:
done